package.json

```diff
@@ -85,7 +86,8 @@
     "prettier": "^2.2.1",
     "sass": "^1.29.0",
     "sass-loader": "^8.0.2",
-    "typescript": "~4.1.5"
+    "typescript": "~4.1.5",
+    "webpack": "^4.1.0"
```

Comment:
Curious on what's the reason to add webpack here 🤔?

Reply:
A dependency  needed webpack to be above 4.1.0 version, without it the building was failing.
![image](https://user-images.githubusercontent.com/2939980/120940287-7efba500-c6ea-11eb-92ac-327e2dd2c142.png)